userspace/falco/configuration.cpp

```diff
@@ -160,6 +160,11 @@ void falco_configuration::merge_configs_files(const std::string& config_name, st
 			}
 		}
 	}
+
+	for(auto &filename : m_loaded_configs_filenames)
```

Comment:
@FedeDP randomly asking you since you worked on this part recently. For `m_loaded_rules_filenames` etc we first clear the list, but we do not do this for the configs filenames list, is this intentional?